scipy/ndimage/tests/test_filters.py

```diff
@@ -911,6 +920,38 @@ def test_filter_axes(self, filter_func, extra_args, size0, size, axes, xp):
         expected = filter_func(array, *extra_args, all_sizes)
         xp_assert_close(output, expected)
 
+    @pytest.mark.parametrize('func', [ndimage.correlate, ndimage.convolve])
+    @pytest.mark.parametrize('dtype', [np.float32, np.complex64])
+    @pytest.mark.parametrize(
+        'axes', tuple(itertools.combinations(range(-3, 3), 2))
+    )
+    @pytest.mark.parametrize('origin', [(0, 0), (-1, 1)])
+    def test_correlate_convolve_axes(self, func, dtype, axes, origin, xp):
+        if is_cupy(xp):
+            pytest.xfail("https://github.com/cupy/cupy/pull/8339")
```

Comment:
Is it important that we `xfail` here rather than `skip`? We don't have `xfail_xp_backends` yet, but we could in theory.

Reply:
Just following the precedent set in gh-21150. It can be resolved in the same follow-up as https://github.com/scipy/scipy/pull/21150/files#r1694245230, right?